problems/isUnique.yaml

```
@@ -0,0 +1,37 @@
+id: 6
+title: IsUnique
+text: |
+  Write a function called isUnique that takes in an array of integers.
+
+  Return true if all the elements of the array are unique.
+  If not, return false.
```

Could you insert an example
```suggestion
  isUnique([1, 2, 3]) = true
  isUnique([1, 1, 5, 5]) = false
```

---

Thanks for the feedback, added some examples!